TEAM.md

```diff
@@ -0,0 +1,56 @@
+# Team membership
+
+Current members are:
+
+- `@alexbit-codemod`
+- `@AugustinMauroy`
+- `@avivkeller`
+- `@bmuenzenmeyer`
+- `@JakobJingleheimer` (lead)
+- `@ljharb`
+
+## Rights
+
+Any team member may block a PR or decision. A block may be overturned by team vote; votes require a quorum of ⅔ of active members and passes by >½ simple majority. In accordance with Node.js policy, a team's vote may be overturned by the [TSC](https://github.com/nodejs/TSC).
+
+Any member can create a tag on main which triggers a CI run, this is used to create a releases. So be careful with this power. Tags should have this structure `vX.Y.Z@workspace`, where `X.Y.Z` is the version number and `workspace` is the name of the npm workspace (e.g. `v1.2.3@codemod`).
+
+### Active vs inactive membership
+
+An active member is one who has substantively participated once in the past 6 weeks.
+
+A member who has been inactive for 4 or more months may be removed. They are welcome to return by simple request.
+
+#### Team lead
+
+The team lead is not more equal than other active members; it is an administrative function. For administrative purposes, the lead owns the repo, the github team, and handles membership changes and repository administration (particularly potentially destructive actions).
+
+The team lead is responsible for ensuring team policies are followed, and takes some discretion when taking administrative action (such as removing an inactive member per team policy).
+
+The team lead can also bypass Github Branch rulesets to merge PRs that are otherwise blocked by the rules. Only for hotfixes.
```

Comment:
```suggestion
The team lead can also bypass Github Branch rulesets to merge PRs that are otherwise blocked by the rules (such as for trivial edits/corrects and hot-fixes).
```